template-manager-cmtemplate/src/test/java/com/sequenceiq/cloudbreak/cmtemplate/configproviders/kudu/KuduServiceConfigProviderTest.java

```diff
@@ -20,17 +20,17 @@
 
 @ExtendWith(MockitoExtension.class)
 class KuduServiceConfigProviderTest {
```

Comment:
If the name change of the tested class is indeed justified, please also rename this UT class.